COMMUNITY.md

```diff
@@ -0,0 +1,40 @@
+# Engaging in the radare2 community
+
+
+## Motivation
+It has turned out that relying on people's common sense is actually not
+sufficient for larger software projects, as there are many different
+interpretation of what this means.
+
+So here is a rather vague explaination of what we'd like to see in the r2
+community (gh issues, source code, irc, telegram, discord, conferences)
+
+## What we don't want to see
+We don't want any form of fascism, racism, sexism, homophobia, transphobia,
+ableism, antisemitism and xenophobia in this community.
+We understand, that people make mistakes, and sometimes don't even know what
+they're saying/doing. This is no excuse for intentional harassment,
+discriminatory and abusive behaviour.
+
+## What we want to see
+We wish to have a supportive community of kind people vibing together,
+connected by common interests, which is radare2 and
```

Comment:
wrong indent, this line is too short

Reply:
i feel like "connected by common interests" should be all in the same line for readability of this document. ... But I might be wrong here

Reply:
> ## What we want to see

# What we would like our community to have

Reply:
> What we don't want to see

"Behaviors we do not allow in our community"

Reply:
> Mild forms of trolling and non-discriminatory profanity is ok for us. Interact with each other as you'd interact with a friend and it will be fine.

"We may understand friendly forms of gestures in trolling or jokes within our community, but what starts as a joke can end up hurting someone, so please always interact with others with respect and in friendly manner."

Reply:
I don't wish to frame it as hard set of rules or laws

Reply:
Well, is not making it a harder set, I just re-phrasing them. That's all.

Reply:
I am serious about this. I want to prevent a situation in which someone comes to us and says "person xyz violated the rules, now you must punish them". This attempts to be more like a community philosophy than a set of rules